src/components/FirstTimeUserFlow/FirstTimeUserFlow.js

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useState, useEffect, createContext } from 'react';
+import { useRouter } from 'next/router';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Paper from '@mui/material/Paper';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import Button from '@mui/material/Button';
+import Typography from '@mui/material/Typography';
+import PersonalInfoForm from './PersonalInfoForm';
+import RoommatePrefForm from './RoommatePrefForm';
+import PropertyPrefForm from './PropertyPrefForm';
+import { postUser } from '@/firebase/userDb';
+
+const steps = ['Personal Information', 'Roommate Preference', 'Property Preference'];
+
+function getStepContent(step) {
+	switch (step) {
+		case 0:
+			return <PersonalInfoForm onFirstNameChange />;
+		case 1:
+			return <RoommatePrefForm />;
+		case 2:
+			return <PropertyPrefForm />;
+		default:
+			throw new Error('Unknown step');
+	}
+}
+
+export default function FirstTimeUserFlow() {
+	const router = useRouter();
+	const [activeStep, setActiveStep] = React.useState(0);
+	const [firstName, setFirstName] = useState('');
+	const [lastName, setLastName] = useState('');
+	const [gender, setGender] = useState('');
+	const [personInfo, setPersonInfo] = useState({});
+
+	const handleNext = () => {
+		setActiveStep(activeStep + 1);
+	};
+
+	const handleBack = () => {
+		setActiveStep(activeStep - 1);
+	};
+
+	const handleSubmit = () => {
+		const person = {
+			'first name': firstName,
+			'last name': lastName,
+		};
+		// postUser('user');
```

Comment:
I'm assuming postUser is a firebase function - is there a reason it's commented out? if we want to leave it commented out, can we leave a "TODO" comment describing what needs to be done for this to get commented back in?

Reply:
Yes. I commented it out because the function is only posting a sample data right now, and I don't want people to post that they are playing around with it. Todo Added.